front_end/helper.py

```diff
@@ -243,3 +244,29 @@ def show_hidden(request_handler):
     if "show_hidden" not in request_handler.request.query_arguments:
         return False
     return request_handler.request.query_arguments["show_hidden"][0].decode().lower() == "true"
+
+def get_dict_of_dates():
+    date_dict = {}
+    years = []
+    months = []
+    days = []
+
+    for i in range(1, 10):
```

Comment:
Please change this to use string formatting. It can add the leading zero for you so that you only need one for loop for days and one for months.